sample-app/src/hooks/useAutocomplete.tsx

```diff
@@ -1,19 +1,30 @@
-import { useRef, useState } from "react";
-import { AutocompleteResult, useAnswersActions } from '@yext/answers-headless-react';
+import { MutableRefObject, useRef, useState } from "react";
+import { AutocompleteResponse, useAnswersActions } from '@yext/answers-headless-react';
 
-export function useAutocomplete(isVertical: boolean): [AutocompleteResult[], () => Promise<void>] {
+export function useAutocomplete(
+  isVertical: boolean
+): [ 
+    AutocompleteResponse|undefined,
+    MutableRefObject<Promise<AutocompleteResponse|undefined>>,
```

Comment:
Would it make sense to have the searchbar keep track of the latest autocomplete response ref? Instead of the hook, since I'm not sure all users of the hook need this. Alternatively I'd probably move it to the end of the returned tuple, since it's more niche than the other 2

Reply:
yea that makes sense. I updated for the function returned from the hook to return the promise so that should be cleaner now!